source/module_base/global_function.h

```diff
@@ -153,11 +153,11 @@ static void READ_VALUE(std::ifstream &ifs, T &v)
     return;
 }
 
-bool SCAN_BEGIN(std::ifstream &ifs, const std::string &TargetName, const bool restart=1);
+bool SCAN_BEGIN(std::ifstream &ifs, const std::string &TargetName, const bool restart=1, const bool ifwarn=true);
 // Mohan warning : the last term can't be written as const bool &restart,
 // I don't know why.
 
-void SCAN_END(std::ifstream &ifs, const std::string &TargetName);
+void SCAN_END(std::ifstream &ifs, const std::string &TargetName, const bool ifwarn=true);
```

Comment:
Please add annotation to describe the parameters of the function.

Reply:
OK, I have added it.